examples/acoustic/acoustic_example.py

```diff
@@ -24,55 +27,94 @@ def source(t, f0):
     return (1-2.*r**2)*np.exp(-r**2)
 
 
-def run(dimensions=(50, 50, 50), spacing=(20.0, 20.0, 20.0), tn=1000.0,
-        time_order=2, space_order=4, nbpml=40, full_run=False, **kwargs):
-
-    origin = (0., 0., 0.)
+def setup(dimensions=(50, 50, 50), spacing=(20.0, 20.0, 20.0), tn=500.,
+          time_order=2, space_order=4, nbpml=10, **kwargs):
 
-    # True velocity
-    true_vp = np.ones(dimensions, dtype=np.float32) + .5
     if len(dimensions) == 2:
-        true_vp[:, int(dimensions[0] / 2):dimensions[0]] = 2.5
-    else:
-        true_vp[:, :, int(dimensions[0] / 2):dimensions[0]] = 2.5
+        # Dimensions in 2D are (x, z)
+        origin = (0., 0.)
+        spacing = (15., 15.)
+
+        # True velocity
+        true_vp = np.ones(dimensions) + .5
+        true_vp[:, int(dimensions[1] / 3):dimensions[1]] = 2.5
+
+        # Source location
+        location = np.zeros((1, 2))
+        location[0, 0] = origin[0] + dimensions[0] * spacing[0] * 0.5
+        location[0, 1] = origin[1] + 2 * spacing[1]
+
+        # Receiver coordinates
+        receiver_coords = np.zeros((dimensions[0], 2))
+        receiver_coords[:, 0] = np.linspace(0, origin[0] +
+                                            (dimensions[0]-1) * spacing[0],
+                                            num=dimensions[0])
+        receiver_coords[:, 1] = location[0, 1]
+
+    elif len(dimensions) == 3:
+        # Dimensions in 3D are (x, y, z)
+        origin = (0., 0., 0.)
+        spacing = (15., 15., 15.)
+
+        # True velocity
+        true_vp = np.ones(dimensions) + .5
+        true_vp[:, :, int(dimensions[2] / 3):dimensions[2]] = 2.5
+
+        # Source location
+        location = np.zeros((1, 3))
+        location[0, 0] = origin[0] + dimensions[0] * spacing[0] * 0.5
+        location[0, 1] = origin[1] + dimensions[1] * spacing[1] * 0.5
+        location[0, 2] = origin[1] + 2 * spacing[2]
+
+        # Receiver coordinates
+        receiver_coords = np.zeros((dimensions[0], 3))
+        receiver_coords[:, 0] = np.linspace(0, origin[0] +
+                                            (dimensions[0] - 1) * spacing[0],
+                                            num=dimensions[0])
+        receiver_coords[:, 1] = origin[1] + dimensions[1] * spacing[1] * 0.5
+        receiver_coords[:, 2] = location[0, 2]
```

Comment:
TTI has ``tti_example2D.py`` and ``tti_example.py`` for 3D.
I think we should be consistent